huaweicloud/services/acceptance/cce/resource_huaweicloud_cce_cluster_test.go

```diff
@@ -661,3 +744,95 @@ resource "huaweicloud_cce_cluster" "test" {
 }
 `, common.TestVpc(rName), rName)
 }
+
+func testAccCluster_resize(rName string) string {
+	return fmt.Sprintf(`
+%s
```

Comment:
change to %[1]s, so that to keep consistent with line 753?

Reply:
sure